starlark/testdata/proto/test.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
```

Comment:
// This proto schema is used for tests in proto.star.
// Re-generate the binary FileDescriptorSet file by 
// running: protoc ...(etc).
// (Requires the "protobuf" brew/apt package.)